docs/workflow/testing/coreclr/gc-stress-run-readme.md

```diff
@@ -14,13 +14,13 @@ This is a pretty crude implementation. Feel free to improve it!
 
 It has 3 parts:
 
-- The stress framework is built from `<REPO_ROOT>\src\coreclr\tests\src\GC\Stress\Framework`
+- The stress framework is built from `<REPO_ROOT>\src\tests\GC\Stress\Framework`
 
-- The tests are built from `<REPO_ROOT>\src\coreclr\tests\src\GC\Stress\Tests`
+- The tests are built from `<REPO_ROOT>\src\tests\GC\Stress\Tests`
 
-- The config is at `<REPO_ROOT>\src\coreclr\tests\src\GC\Stress\testmix_gc.config`, this will be copied to the output folder of Framework
+- The config is at `<REPO_ROOT>\src\tests\GC\Stress\testmix_gc.config`, this will be copied to the output folder of Framework
 
-The easiest way to build the Framework+Tests is by building all tests - `<REPO_ROOT>\src\coreclr\build_test[.bat|.sh]`
+The easiest way to build the Framework+Tests is by running `dotnet msbuild` from `<REPO_ROOT>` on `src\tests\GC\Stress\Framework\ReliabilityFramework.csproj`.
 
 Sometimes there is a need (after initial build) to rebuild Framework+Tests. For example when modifying the Framework to add a new scenario or when investigating a failure.
```

Comment:
this line should probably be removed.. if you change code in Framework, I'd think everyone knows to rebuild it 😀 so they can use the cmdline right above this.